tasks/secrets_protection.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Add Confluent CLI to path
```

Comment:
i would just change the below shell commands to `/usr/local/bin/confluent secret` and remove this task